libs/partners/ollama/langchain_ollama/chat_models.py

```diff
@@ -513,25 +514,26 @@ def _create_chat_stream(
                 params[key] = kwargs[key]
 
         params["options"]["stop"] = stop
-        if "tools" in kwargs:
-            yield self._client.chat(
-                model=params["model"],
-                messages=ollama_messages,
-                stream=False,
-                options=Options(**params["options"]),
-                keep_alive=params["keep_alive"],
-                format=params["format"],
-                tools=kwargs["tools"],
-            )
+
+        tools = kwargs.get("tools", None)
```

Comment:
Hello, I've added the change you asked for to simplify this part of the code.

Reply:
Thank you!